### PR TITLE
Make untangle available from conda-forge

### LINKED ISSUE #72: Make untangle installable from conda-forge?
Hi @stchris!

Thanks for creating this very useful package! I use it myself in here: https://github.com/HTenkanen/transx2gtfs

I was just creating a conda-forge recipe for my library, and realized that `untangle` is not currently available from conda. 

Would you be okay with the idea of adding untangle to conda-forge? The thing with conda is, that they are very restrictive that all dependencies of libraries need to come from conda-forge (in this way the reliability of the whole system is ensured). Hence, any libraries that depends on untangle cannot be published in conda-forge easily (such as mine).

If you are open to this idea, I am happy to help with writing a conda-forge recipe for untangle. The process does not require any modifications to the current repo, but the things are done by forking the "conda-forge/staged-recipes" repo.

More information about the whole conda process can be found from here:
https://conda-forge.org/

### PR DESCRIPTION
**Pull request by HTenkanen**
Closes #72 

Untangle has been added to conda-forge feedstock. Adds documentation about how to install with conda.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,20 @@ untangle
 Installation
 ------------
 
+Untangle can be easily installed with pip and conda package managers. 
+
+Installation with pip:
```
Comment (stchris) [inline README.md:22]:
```suggestion
With pip:
```

File: README.md [hunk 2]
```diff
@@ -17,10 +17,20 @@ untangle
...
 ```
 pip install untangle
 ```
 
+Installation with conda:
```
Comment (stchris) [inline README.md:27]:
```suggestion
With conda:
```

File: README.md [hunk 3]
```diff
@@ -17,10 +17,20 @@ untangle
...
+```
+conda install -c conda-forge untangle
+```
+
+
```
Comment (stchris) [inline README.md:32]:
```suggestion
```

File: docs/index.rst [hunk 4]
```diff
@@ -54,7 +54,11 @@ It is recommended to use pip, which will always download the latest stable relea
 
     pip install untangle
 
-untangle works with Python versions 2.6, 2.7, 3.3, 3.4, 3.5, 3.6 and pypy
+untangle works with Python versions 2.6, 2.7, 3.3, 3.4, 3.5, 3.6 and pypy.
+
+Alternatively, you can install untangle with conda-forge: ::
```
Comment (stchris) [inline docs/index.rst:59]:
```suggestion
Alternatively, you can install untangle with conda from conda-forge: ::
```

**Comment by HTenkanen** [comment]
@stchris Okay, added comment about conda maintenance to the readme and added myself to the contributors.

**Comment by stchris** [comment]
LGTM!

**Pull request merged**